examples/kubernetes/connectivity-check/Makefile

```diff
@@ -28,24 +30,28 @@ all: $(ALL_TARGETS)
 
 $(DEFAULT_OUT): $(SRC)
 	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(DEFAULT_OUT)
-	$(QUIET)$(CUE) dump > $(DEFAULT_OUT)
+	$(QUIET)$(CUE) dump >> $@
+
+$(IPv6_ONLY_OUT): $(SRC)
+	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(IPv6_ONLY_OUT)
+	$(QUIET)$(CUE) -t ipFamily=IPv6 dump >> $@
 
 $(HOSTPORT_OUT): $(SRC)
 	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(HOSTPORT_OUT)
-	$(QUIET)$(CUE) -t component=all dump > $(HOSTPORT_OUT)
+	$(QUIET)$(CUE) -t component=all dump >> $@
```

Comment:
For reviewer, there was a mixed usage of $@ and $(OUT_FILE) in this Makefile. As I am changing this file, I just change all to $@ for standardization.

$@
The file name of the target of the rule. If the target is an archive member, then ‘$@’ is the name of the archive file. In a pattern rule that has multiple targets (see Introduction to Pattern Rules), ‘$@’ is the name of whichever target caused the rule’s recipe to be run.

Reply:
While we are at this, I am not sure why there is an inconsistent usage of `@` and `$(QUIET)` in the Makefile. Is there a particular reason we are using a mix of `@` and `$(QUIET)` here @joestringer ?

Reply:
I think we can control the baviour based of `$(QUIET)` based on $(V) (e.g. versbose), meanwhile, `@` behavior is unchangable.

Reply:
But I still don't understand if a mix of these makes sense here. :sweat_smile:
Anyways the changes look good to me overall and this is not a problem.

Reply:
Re: inconsistent usage, for some cases I figure that when you want to debug or understand the process occurring, you would want to see the commands being run. Typically for `echo` commands (especially when printing output to the terminal), this is not useful because you end up with duplicate printing: Once in the command output and once in the stdout.

Though that said I almost exclusively turn $(QUIET) on so I forget what the build looks like without it :-)